.rubocop.yml

```diff
@@ -5,6 +5,23 @@ AllCops:
     - vendor/**/*
     - gemfiles/**/*
   TargetRubyVersion: 2.4
+  NewCops: enable
+
+
+################
+# Layout Rules #
+################
+Layout/EmptyLinesAroundClassBody:
+  EnforcedStyle: empty_lines_except_namespace
+
+Layout/EmptyLinesAroundModuleBody:
+  EnforcedStyle: empty_lines_except_namespace
+
+Layout/LineLength:
+  Max: 140
+
+Layout/SpaceInsideHashLiteralBraces:
+  EnforcedStyle: space
```

Comment:
They are not new, but were renamed or have a different category (`Layout` instead of `Style`) between RuboCop 0.47 and 0.82.